src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random colorIndex = new Random();
```

Comment:
```suggestion
    private Random random = new Random();
```

не информативное название переменной